spec/features/results_spec.rb

```diff
@@ -0,0 +1,6 @@
+feature 'Lets the player know whos won' do
+  scenario 'The player beats the Computer' do
+    visit('/')
```

Comment:
Some of your feature tests visit the root page, wondering if you would need the test to cycle through all the routes to get the right page